src/utils.ts

```diff
@@ -44,9 +44,11 @@ const waitUntil = async (condition: () => Promise<boolean>, isNot = false, {
             throw error
         }
 
-        return !isNot
+        // condition was fufilled after wait
+        return true
     } catch (err) {
-        return isNot
+        // condition was not fufilled after wait
+        return false
```

Comment:
I believe this is causing the mock matcher tests to fail. I would recommend to keep as is.

Reply:
I have removed this change and the tests now